.coveragerc

```diff
@@ -2,3 +2,7 @@
 omit = */tests/*
     cumulusci/files/*
 source=cumulusci
+[report]
+exclude_lines =
+    pragma: no cover
```

Comment:
We currently use "pragma: no cover" in 1 place and "pragma: nocover" in 5 places. Can you either make this exclude both of them, or update them to be consistent?